inference-engine/src/inference_engine/include/ie/ie_remote_context.hpp

```diff
@@ -110,6 +110,14 @@ class RemoteContext : public std::enable_shared_from_this<RemoteContext> {
      */
     virtual RemoteBlob::Ptr CreateBlob(const TensorDesc& tensorDesc, const ParamMap& params = {}) = 0;
 
+    /**
+     * @brief Allocates host accesible memory blob
+     * Returns a pointer to the object which implements MemoryBlob interface.
+     * @param tensorDesc Defines the layout and dims of the blob
+     * @return A pointer to host accesible MemoryBlob object
```

Comment:
`accesible` -> `accessible`

Can we also add a note that it's device friendly blob?